CHANGELOG.md

```diff
@@ -18,7 +18,11 @@ Please follow the recommendations outlined at [keepachangelog.com](http://keepac
 ### [Unreleased]
 Changes since the last non-beta release.
 
-#### Added(https://github.com/AbanoubGhadban).
+#### Fixed
+
+- Changed the ReactOnRails' version checker to check package.jsons in both root and client directories. [PR 1657](https://github.com/shakacode/react_on_rails/pull/1657) by [judahmeek](https://github.com/judahmeek).
```

Comment:
Should be using `node_modules_location`.

https://github.com/search?q=repo%3Ashakacode%2Freact_on_rails%20node_modules_location&type=code